### ISSUE TITLE
Change the output format of predict.py

### ISSUE THREAD

**Issue by s-jse** [stanford-oval/genienlp#60]
We support multiple predictions per input, and soon we will add a confidence score for each prediction. Keeping the `tsv` format will make things difficult down the line. I propose using `jsonl` instead.

**Comment by gcampax** [comment]
jsonl means one JSON blob per line right? I don't mind. One of the requirements is being able to concatenate files, and process files line by line in a streaming fashion, so this will still work. A single JSON blob on the other hand would not work, because Python has no streaming JSON library.

**Comment by Mehrad0711** [comment]
Yes. I'm much in favor of it. More intuitive to index, plus bootleg uses jsonl so I can drop the conversion between jsonl and tsv.
